generate_export_options.rb

```diff
@@ -98,6 +98,7 @@ def export_method(mobileprovision_content)
 
 export_options = {}
 export_options[:method] = method unless method.nil?
+export_options[:uploadBitcode] = false
 
```

Comment:
This should be optional at least - there are certain type of projects which you can't distribute with bitcode disabled!